docs/reference/openapi-rules.md

```diff
@@ -1,3 +1,546 @@
-## OpenAPI Rules
+# OpenAPI Rules
 
-Coming soon...
\ No newline at end of file
+Spectral has three rulesets built in: 
+
+- [oas](#oas)
+- [oas2](#oas2)
+- [oas3](#oas3)
+
+_OAS is shorthand for OpenAPI Specification._
+
+In your ruleset file you can add `extends: "spectral:oas"` and you'll get everything from `spectral:oas2` and `spectral:oas3`, meaning you can lint either type of document and only the relevant rules will apply thanks to [formats](../getting-started/rulesets.md#formats).
+
+Let's look at the rules in these rulesets.
+
+## oas
+
+### operation-2xx-response
+
+Operation must have at least one `2xx` response. Any API operation (endpoint) can fail but presumably it is also meant to do something constructive at some point. If you forget to write out a success case for this API, then this rule will let you know.
+
+**Recommended:** Yes
+
+**Good Example**
```

Comment:
Hey, how about 
```suggestion
**Will fail validation**
```
Or something similar? Doesn't "good example" sound like the following code snippet is "valid"?

Reply:
Copy paste error. Good Example does indeed mean the code is valid.